### PR TITLE
updated banner to add caveat about RC state

### PR DESCRIPTION
**Pull request by zr-msft**
Proposed changes to banner to add clarity around state of notation cli and docs

### CODE REVIEW

File: layouts/partials/banner.html [hunk 1]
```diff
@@ -1,19 +1,13 @@
 <section class="news is-medium flex justify-center">
-  <p class="is-size-3-mobile">
-    <span>
-      {{ $rocketIcon := resources.Get "icons/rocket-icon.svg" }}
-      <img
+    <p class="is-size-3-mobile">
+      <a href="https://github.com/notaryproject/notation/releases/tag/v1.0.0-rc.3" class="banner-link">Notation v1.0.0-RC.3 is available</a>
```
Comment (yizha1) [inline layouts/partials/banner.html:3]: Since rc.4 is coming, suggest changing the banner to rc.4

Comment (FeynmanZhou) [reply layouts/partials/banner.html:3]: Agree with @zr-msft . We could send another PR to update it to RC.4 after the RC.4 is out.

**Review by FeynmanZhou — APPROVED** [review]
LGTM. It adds clarity to the RC status of the Notation product and documentation

**Review by yizha1 — APPROVED** [review]
LGTM

**Pull request merged**